components/modules/NewReleases/index.tsx

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable @next/next/no-img-element */
+import React, { useState } from "react";
+import { songs } from "../utils/dummy";
+import { VDots, VPause, VPlay, VStar, VWave } from "../__modules__/_Vectors";
+import liveBg from "../../../public/livebg.jpg";
+import AudioPlayer from "../utils/config/audioPlayer";
+import equalizer from "../static/images/equalizer.gif";
+
+interface TrackItems{
+  [index: number]:{title: string;
+  artist: string;
+  audioSrc: string;
+  image: string;}
+}
```

Comment:
As this is a type of mock data that will be fetched from the API after, it may be good to create a `type` folder and separate it from the component.

Reply:
got it, lemme clear it up